src/languages/en.ts

```diff
@@ -819,11 +819,9 @@ export default {
             genericHoldExpenseFailureMessage: 'Unexpected error holding this expense. Please try again later.',
             genericUnholdExpenseFailureMessage: 'Unexpected error taking this expense off hold. Please try again later.',
             receiptDeleteFailureError: 'Unexpected error deleting this receipt. Please try again later.',
-            // eslint-disable-next-line rulesdir/use-periods-for-error-messages
-            receiptFailureMessage: "The receipt didn't upload. ",
-            // eslint-disable-next-line rulesdir/use-periods-for-error-messages
+            receiptFailureMessage: "The receipt didn't upload",
```

Comment:
I think we should keep this as it is as the error seems to be used in phrases like - `The receipt didn't upload. Download the file or dismiss this error and lose it.`

Reply:
@jamesdeanexpensify what do you think?

Reply:
Oh, got it! Sounds good, Monil.